### PR TITLE
Ensure provenance relations are used with ORCID or other appropriate IRIs

### PR DESCRIPTION
**Pull request by cthoyt**
This check should ensure that `IAO:0000117` (term editor), `dc:contributor`, `dc:creator`, and `oio:created_by` all have appropriate ORCID/Wikidata/ROR IRIs. I also did all of the curation it took to pass this check.

This is based on @anitacaron's suggestion from UBERON/CL: https://github.com/obophenotype/uberon/blob/master/src/sparql/orcid-contributor-violation.sparql

### CODE REVIEW

File: src/ontology/Makefile [hunk 1]
```diff
@@ -39,10 +39,10 @@ SPARQLDIR =                 ../sparql
 COMPONENTSDIR =             components
 ROBOT_PROFILE =             profile.txt
 REPORT_FAIL_ON =            ERROR
-REPORT_LABEL =              
+REPORT_LABEL =              -l true
 REPORT_PROFILE_OPTS =       --profile $(ROBOT_PROFILE)
 OBO_FORMAT_OPTIONS =        
-SPARQL_VALIDATION_CHECKS =  owldef-self-reference iri-range label-with-iri multiple-replaced_by
```
Comment (cthoyt) [inline src/ontology/Makefile:45]: not sure why these got thrown away, were these like a default profile?

Comment (matentzn) [reply src/ontology/Makefile:45]: Its because you decided to customise the list - these were the defaults when you do not customise the list of checks!

Comment (cthoyt) [reply src/ontology/Makefile:45]: hmm okay so I will just re-capitulate the list here into the "custom" list so they don't get thrown away.

**Comment by cmungall** [comment]
Thanks @cthoyt !

This makes `ro statistics` look much cleaner:

```yaml
...
  orcid:0000-0002-6601-2165:
    contributor_id: orcid:0000-0002-6601-2165
    role_counts:
      IAO:0000117:
        facet: IAO:0000117
        filtered_count: 260
      dcterms:contributor:
        facet: dcterms:contributor
        filtered_count: 2
      dcterms:creator:
        facet: dcterms:creator
        filtered_count: 7
      oio:created_by:
        facet: oio:created_by
        filtered_count: 46
  orcid:0000-0002-7073-9172:
    contributor_id: orcid:0000-0002-7073-9172
    role_counts:
      IAO:0000117:
        facet: IAO:0000117
        filtered_count: 38
      dcterms:contributor:
        facet: dcterms:contributor
        filtered_count: 1
      oio:created_by:
        facet: oio:created_by
        filtered_count: 15
...
```


I would like to also normalize on dcterms:contributor (see https://github.com/information-artifact-ontology/ontology-metadata/issues/120 where I was convinced of this). But this can be a separate PR! onwards!

**Review by cmungall — APPROVED** [review]

**Review by matentzn — APPROVED** [review]

**Pull request merged**